app/data/contentscript.js

```diff
@@ -3,31 +3,31 @@
 
 var pathComponents = window.location.pathname.split('/');
 var _logName = 'github-forks-extension:';
-var lovelyForksClass = 'lovely-forks-addon-class';
 var DEBUG = false;
+var el = new WeakMap();
```

Comment:
Why have a `WeakMap` here at all if just a `var el = null;` appears sufficient?
Then, while trying to retrieve the element, we'dd be able to write `var text = el || createElem()`.

Support for WeakMap seems sufficient, but I am a little edgy about using it for old versions of Firefox and for Firefox@Android.